realis.go

```diff
@@ -542,13 +503,15 @@ func (r *realisClient) GetInstanceIds(key *aurora.JobKey, states []aurora.Schedu
 
 	r.logger.DebugPrintf("GetTasksWithoutConfigs Thrift Payload: %+v\n", taskQ)
 
-	resp, retryErr := r.thriftCallWithRetries(func() (*aurora.Response, error) {
-		return r.client.GetTasksWithoutConfigs(nil, taskQ)
-	})
+	resp, retryErr := r.thriftCallWithRetries(
```

Comment:
This retry and retryErr is almost same in all the func ... though I am not sure if there are better ways because all of them have a few difference in message and arguments

Reply:
Yeah it's tough to make this match all functions. This should have really been like what it's like in V2 all along.